src/models/component.class.ts

```diff
@@ -90,64 +95,58 @@ export abstract class Component {
                 this.listen(this[el.propertyKey], el.eventType, this[el.handlerFnName]);
             }
         });
-        
+
         // User Hook
         this.load();
     }
 
-    /* Hooks */
-    load() {
-        // Placeholder hook for inherited classes
-    }
-
-    append(parent?: Node, doNotLoad?: boolean) {
-        if (!this.parsedNode) {
-            this.createNode();
-        }
-
-        if (!parent) {
-            parent = document.body;
-        }
-        parent.appendChild(this.parsedNode);
-
-        // Opt out of loading if this is an ingredient
-        if (doNotLoad) {
-            return;
-        }
-
-        // Run load all, which loads children, then the load hook
-        this.loadAll(parent);
-    }
-
     redraw() {
-        if (this.element) {
-            const newNode = ParseEngine.parseNode(this.ogNode, this.data);
-            const oldNode = document.getElementById(this.id);
-            this.parent.replaceChild(newNode, oldNode);
-            this.parsedNode = newNode;
-            this.element = document.getElementById(this.id);
+        // Remove 
+        const newParse = ParseEngine.parseNode(this.ogNode, this.data);
+        const newEl = newParse.el;
+        const recipe = newParse.recipe;
```

Comment:
Recipe is just left hanging. This needs to somehow replace the previous children.

Reply:
This is being left as it's own issue rather than dragging it into this one.